### ISSUE TITLE
Truncated text (numbers) in Stats widget on xperia Z5 Compact

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#3496]
![screenshot_2015-12-10-10-42-16](https://cloud.githubusercontent.com/assets/40213/11712097/2a9cbaea-9f2b-11e5-98e1-59f242cf2e6d.png)

**Comment by daniloercoli** [comment]
It's not easy to give the widget a fixed height in `dp` (to match one row), and then use `sp` in text.
Need to find a cool way to handle this issues...maybe an auto-layout thing of some sort. Anyway the widget usage is too low, that i'd like to postpone this until we find a nice way to promote the widget.
(We're working on a way of promoting this widget when the user open Stats).

**Comment by daniloercoli** [comment]
I've investigated if we can fix this kind of issues once and for all. This current issue is strictly related to another issue that could happen when the user sets the "Huge" font size in device settings.

Unfortunately doesn't seem to be there an easy fix. I've installed other apps, and tried their widgets, and the only way for a "fixed" screen (no scrolling) widget is to use dp. See pictures below:

* Huge font size: 
![screen shot 2015-12-17 at 15 41 54](https://cloud.githubusercontent.com/assets/518232/11875200/da7a10a8-a4e3-11e5-810e-af522cf0930b.png)
* Normal font size:
![screen shot 2015-12-17 at 15 42 08](https://cloud.githubusercontent.com/assets/518232/11875202/dd8b2ea8-a4e3-11e5-99e0-f1560d5fcdfc.png)

* Huge font size: 
![screen shot 2015-12-17 at 15 29 08](https://cloud.githubusercontent.com/assets/518232/11875249/17697e7c-a4e4-11e5-9950-31d5bbe15893.png)

* Normal font size:
![screen shot 2015-12-17 at 15 29 28](https://cloud.githubusercontent.com/assets/518232/11875260/26ee9ee0-a4e4-11e5-82a2-3dc8f55c31e1.png)

@maxme and others, Is it ok for you if I use `dp` to specify font size?
